### PR TITLE
Upgrade to androidx and Android Studio 3.6.x

### PR DESCRIPTION
**Pull request by 3c71**
That way sample is up-to-date with latest Android development, eg primarily AndroidX

### CODE REVIEW

**Comment by greg-db** [comment]
Thanks! It looks like there is some overlap with this recent PR though: https://github.com/dropbox/dropbox-sdk-java/pull/271

File: examples/android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
             <intent-filter>
 
                 <!-- Change this to be db- followed by your app key -->
-                <data android:scheme="db-YOUR_APP_KEY_HERE" />
+                <data android:scheme="db-yesvjbrhv075ttz" />
```
Comment (greg-db) [inline examples/android/src/main/AndroidManifest.xml:33]: Looks like you accidentally left your app key in. You should remove this.

Comment (3c71) [reply examples/android/src/main/AndroidManifest.xml]: oops, obsolete key though. I've commit'd both files affected (strings.xml had it too). Is it automatically included in PR?

**Comment by 3c71** [comment]
> Thanks! It looks like there is some overlap with this recent PR though: #271

Ah! Yes didn't see that. Either PR will get sample up-to-date then.

**Pull request merged**